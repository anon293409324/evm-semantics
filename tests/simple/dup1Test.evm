account:
-   id: 1
-   balance: 40
-   program: PUSH 40 ; PUSH 50 ; DUP 2 ; DUP 3
-   storage: 30 , 20 , 25

.Accounts


transaction:
-   to: 1
-   from: .AcctID
-   data:
-   value: 0
-   gasPrice: 1
-   gasLimit: 100

.Transactions
