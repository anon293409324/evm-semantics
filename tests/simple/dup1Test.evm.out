<T> <k> . </k> <id> 1 </id> <pc> 4 </pc> <gas> 88 </gas> <wordStack> 40 : ( 40 : ( 50 : ( 40 : .WordStack ) ) ) </wordStack> <localMem> .Map </localMem> <gasPrice> 1 </gasPrice> <callStack> .CallStack </callStack> <accounts> <account> <acctID> 1 </acctID> <program> 0 |-> ( PUSH 40 ) 1 |-> ( PUSH 50 ) 2 |-> ( DUP 2 ) 3 |-> DUP 3 </program> <storage> 0 |-> 30 1 |-> 20 2 |-> 25 </storage> <balance> 40 </balance> </account> </accounts> </T>
